src/code/code_80106860.c

```diff
@@ -0,0 +1,13 @@
+#include <ultra64.h>
+#include <global.h>
+
+// memset used in __osMalloc, z_quake, z_view, and z_camera
+void *func_80106860(void *ptr, s32 val, size_t size) {
```

Comment:
the style in this codebase is star for pointers on the left. clang will catch this if you run your files through format.sh